File: testers/grpc/kotlin/generated-and-checked-in/com/example/grpc/OrderId.kt
package com.example.grpc



/** Wrapper type for typr.grpc.GrpcCodegen$$$Lambda/0x00007fc001118400@5884a914 */
data class OrderId(val value: kotlin.String) {
  /** Get the underlying value */
  fun unwrap(): kotlin.String {
    return this.value
  }

  override fun toString(): kotlin.String {
    return value
  }

  companion object {
    /** Create a OrderId from a raw value */
    fun valueOf(v: kotlin.String): OrderId {
      return OrderId(v)
    }
  }
}

File: testers/grpc/scala/generated-and-checked-in/com/example/grpc/OrderId.scala
package com.example.grpc



/** Wrapper type for typr.grpc.GrpcCodegen$$$Lambda/0x00007fc001118400@65987993 */
case class OrderId(value: String) extends scala.AnyVal {
  /** Get the underlying value */
  def unwrap: String = {
    return this.value
  }
}

object OrderId {
  /** Create a OrderId from a raw value */
  def valueOf(v: String): OrderId = {
    return new OrderId(v)
  }
}